ssg/build_yaml.py

```diff
@@ -1374,6 +1376,15 @@ def normalize(self, product):
             )
             raise RuntimeError(msg)
 
+    def add_stig_references(self, stig_references):
+        stig_id = self.references.get("stigid", None)
+        if not stig_id:
+            return
+        reference = stig_references.get(stig_id, None)
+        if not reference:
+            return
+        self.references["stigref"] = reference
```

Comment:
Question: Do you think that the `stigref` and `stigid` strings would be worth promoting to constants?

Reply:
I don't think so, they are different for each rule.

Reply:
I meant the literal `stigref` string. But after discussing and thinking about it, it wouldn't bring significant added value to the project.